tideways/tideways-header.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-if( isset( $_SERVER['REQUEST_URI'] ) && strpos( $_SERVER['REQUEST_URI'], 'enable-tideways' ) !== false ) {
+if( file_exists( '/srv/tideways.json' ) && in_array( $_SERVER['HTTP_HOST'], json_decode( file_get_contents( '/srv/tideways.json' ) ) ) || isset( $_SERVER['REQUEST_URI'] ) && strpos( $_SERVER['REQUEST_URI'], 'enable-tideways' ) !== false ) {
```

Comment:
This is getting long! lets break it up and start returning early from the file, e.g.

```php
if ( !file_exists( '/srv/tideways.json' ) ) {
    return;
}
```